pkg/signup/service/signup_service.go

```diff
@@ -431,7 +431,9 @@ func (s *ServiceImpl) DoGetSignup(ctx *gin.Context, provider ResourceProvider, u
 		return signupResponse, nil
 	}
 
-	if completeCondition.Status != apiv1.ConditionTrue {
+	// in proxy, we don't care if the usersignup is completed, since sometimes it might be transitioning from complete to provisioning
+	// which cases issues with some proxy calls.
```

Comment:
It can be even useful to add a link to the corresponding JIRA with the detailed explanation of the problem we are solving by introducing it.

Reply:
done plz see: https://github.com/codeready-toolchain/registration-service/pull/345/commits/73dd603dac216e87d78b8f1d22905232caedf0b8